contracts/main/stablecoin-core/LiquidationEngine.sol

```diff
@@ -38,14 +38,15 @@ contract LiquidationEngine is PausableUpgradeable, ReentrancyGuardUpgradeable, I
     // --- Math ---
     uint256 internal constant WAD = 10 ** 18;
 
-    bytes32 internal deprecated;
-
     IBookKeeper public bookKeeper; // CDP Engine
     ISystemDebtEngine public systemDebtEngine; // Debt Engine
     uint256 public override live; // Active Flag
     mapping(address => bool) public liquidatorsWhitelist;
 
     event LiquidationFail(bytes32 _collateralPoolIds, address _positionAddresses, address _liquidator, string reason);
+    event LogSetBookKeeper(address indexed _newAddress);
```

Comment:
why to keep it indexed? you kept access control address non-indexed in its event

Reply:
Actually, no need. Valid point. newAddress for bookKeper is not in critical need for later search. Will make it un indexed.